### PR TITLE
RELATED: RAIL-3586 improve integration docs

### PR DESCRIPTION
**Pull request by no23reason**
Mention how to setup the app in a cross-domain use case.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/786/ : SUCCESS in 1m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/503/ : SUCCESS in 8s

File: docs/02_start__no_boilerplate.md [hunk 1]
```diff
@@ -121,7 +126,10 @@ This is how you can trigger the username and password login process using `@good
 
 ```javascript
 import { factory } from "@gooddata/api-client-bear";
+
 const bearClient = factory();
+// or if your application will be hosted on a different host than the GoodData platform backend
+const bearClient = factory({ domain: "https://example.com" }); // this should be the domain the GoodData platform is hosted
```
Comment (evgeniabogranskaya) [inline docs/02_start__no_boilerplate.md:132]: this should be the domain where the GoodData platform is hosted

File: docs/06_cloudnative__integration.md [hunk 2]
```diff
@@ -67,9 +67,14 @@ Make sure to import the styles only from the packages that you actually use.
...
 
 ```javascript
-import tigerFactory, {ContextDeferredAuthProvider, redirectToTigerAuthentication} from "@gooddata/sdk-backend-tiger";
+import tigerFactory, { ContextDeferredAuthProvider, redirectToTigerAuthentication } from "@gooddata/sdk-backend-tiger";
 
 const backend = tigerFactory().withAuthentication(new ContextDeferredAuthProvider(redirectToTigerAuthentication));
+
+// or if your application will be hosted on a different host than the GoodData.CN backend
+const backend = tigerFactory()
+    .onHostname("https://example.com") // this should be the domain the GoodData.CN is hosted
```
Comment (evgeniabogranskaya) [inline docs/06_cloudnative__integration.md:76]: this should be the domain where GoodData.CN is hosted

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/788/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/505/ : SUCCESS in 7s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR439

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/790/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/140/ : SUCCESS in 54s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/40/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/230/ : SUCCESS in 17s

**Pull request merged**